onnx/defs/schema.h

```diff
@@ -257,6 +265,8 @@ class OpSchema {
   int min_output_ = 0;
   int max_output_ = std::numeric_limits<int>::max();
   std::set<int> optional_inputs_;
+  // The default is a little goofy, since it is never what you want
```

Comment:
make it -1 and you can enforce that it's set in Finalize() call. But then you'd need to codemod all operator defs